packages/react-native-web/src/modules/useElementLayout/index.js

```diff
@@ -29,20 +29,23 @@ function getResizeObserver(): ?ResizeObserver {
           if (typeof onLayout === 'function') {
             // We still need to measure the view because browsers don't yet provide
             // border-box dimensions in the entry
-            UIManager.measure(node, (x, y, width, height, left, top) => {
-              const event: LayoutEvent = {
-                // $FlowFixMe
-                nativeEvent: {
-                  layout: { x, y, width, height, left, top }
-                },
-                timeStamp: Date.now()
-              };
-              Object.defineProperty(event.nativeEvent, 'target', {
-                enumerable: true,
-                get: () => entry.target
-              });
-              onLayout(event);
-            });
+            UIManager.measure(
+              node,
+              (x, y, width, height, left, top, offsetTop) => {
+                const event: LayoutEvent = {
+                  // $FlowFixMe
+                  nativeEvent: {
+                    layout: { x, y, width, height, left, top, offsetTop }
```

Comment:
Again, this is changing the expected values from the RN API and might be a problem if users start to think they can depend on `offsetTop` across platforms.